skills/stopwatch.js

```diff
@@ -0,0 +1,171 @@
+const schedule = require('node-schedule');
```

Comment:
can you run eslint on this file? also please make sure any vars are converted to either `const` or `let`